internal/storage/mocks/mock_dynamodb.go

```diff
@@ -81,6 +85,30 @@ func (mdb *DynamoDB) DeleteItem(_ context.Context, in *dynamodb.DeleteItemInput,
 	return mdb.DeleteItemOut, mdb.DeleteItemError
 }
 
-func (mdb *DynamoDB) Scan(_ context.Context, in *dynamodb.ScanInput, _ ...func(*dynamodb.Options)) (*dynamodb.ScanOutput, error) {
-	panic("Not implemented")
+func (mdb *DynamoDB) Scan(ctx context.Context, in *dynamodb.ScanInput, optFns ...func(*dynamodb.Options)) (*dynamodb.ScanOutput, error) {
+
+	mdb.ScanItemInvocations++
+	mdb.TableName = *in.TableName
+
+	mdb.ScanOut = &dynamodb.ScanOutput{
+		Items: []map[string]types.AttributeValue{
+			{
+				"Version": &types.AttributeValueMemberS{Value: "1.0.1"},
+			},
+			{
+				"Version": &types.AttributeValueMemberS{Value: "1.0.10"},
+			},
+			{
+				"Version": &types.AttributeValueMemberS{Value: "1.0.2-beta"},
+			},
+			{
+				"Version": &types.AttributeValueMemberS{Value: "1.0.2-alpha"},
+			},
+			{
+				"Version": &types.AttributeValueMemberS{Value: "v1.0.2-gamma"},
+			},
+		},
+	}
```

Comment:
This is the wrong place for this, when you're creating the mock DynamoDB object you pass this into the new Mock object so that it can return this.
```suggestion
```

Reply:
Thanks, have addressed this. @adcharre